numpy/core/tests/test_multiarray.py

```diff
@@ -5497,10 +5498,12 @@ def test_binary(self, tmp_filename):
 
     @pytest.mark.slow  # takes > 1 minute on mechanical hard drive
     def test_big_binary(self):
-        """Test workarounds for 32-bit limited fwrite, fseek, and ftell
-        calls in windows. These normally would hang doing something like this.
-        See http://projects.scipy.org/numpy/ticket/1660"""
-        if sys.platform != 'win32':
+        """Test workarounds for 32-bit limit for MSVC fwrite, fseek, and ftell
+
+        These normally would hang doing something like this.
+        See : https://github.com/numpy/numpy/issues/2256
+        """
+        if sys.platform != 'win32' or re.search(r'\[GCC ', sys.version):
```

Comment:
```suggestion
        if sys.platform != 'win32' or '[GCC ' in sys.version:
```
Feels easier on the eye.